lib/routes/aimeta/blog.ts

```diff
@@ -0,0 +1,36 @@
+import { Route } from '@/types';
+import ofetch from '@/utils/ofetch';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+
+export const route: Route = {
+    path: '/blog',
+    categories: ['blog', 'programming'],
+    example: '/aimeta/blog',
+    radar: [{ source: ['ai.meta.com/blog'] }],
+    name: 'Blog',
+    maintainers: ['canonnizq'],
+    handler
+};
+
+async function handler() {
+    const baseUrl = 'https://ai.meta.com';
+
+    const response = await ofetch(`${baseUrl}/blog/`);
+    const $ = load(response);
+
+    const items = $('div._ams_')
+        .toArray().map((item) => ({
+            category: $(item).find('p._amt0').text(),
+            link: $(item).find('a._amt1').attr('href'),
+            title: $(item).find('a._amt1 p._amt2').text(),
+            description: $(item).find('div._4ik4._4ik5 p._amt3').text(),
```

Comment:
`_4ik4` and `_4ik5` are class name generated from CSS in JS and will be altered in future updates.